PED/GetPedDrawableVariation.md

```diff
@@ -17,7 +17,7 @@ Ids
 4 - Legs
 5 - Hands
 6 - Foot
-7 - ------
+7 - Scarfs/Neck Accessories
 8 - Accessories 1
 9 - Accessories 2
 10- Decals
```

Comment:
Huh, there's one more apparently, `11` seems to be `JBIB`, which is used for torso suits/jackets

```c
if ( drawableVariationIdx > 6 )
    {
      switch ( drawableVariationIdx )
      {
        case 7:
          v7 = "TEETH";
          break;
        case 8:
          v7 = "ACCESSORIES";
          break;
        case 9:
          v7 = "TASK";
          break;
        case 10:
          v7 = "DECL";
          break;
        default: // Most likely 11
          v7 = "JBIB";
          break;
      }
```

You don't have to add it though, I could add it at a later PR 🙂

Reply:
11 is jackets. Some of the other ones might be wrong as well, I was making a clothing customisation menu and was researching all of the drawables and props. So I have a list of all the correct ones.

Reply:
> 11 is jackets. Some of the other ones might be wrong as well, I was making a clothing customisation menu and was researching all of the drawables and props. So I have a list of all the correct ones on my computer, won't be available for the weekend tho...

Yeah, I searched for "JBIB" in the rpf files and figured it had something to do with jackets as you are saying. Anyway, that's ok. Have a great weekend!

Reply:
Yep. You too 😄